dydx-testnet-4/gentx/gentx-dydx-stakelab.json

```diff
@@ -0,0 +1,2 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"StakeLab","identity":"F12B081334CBE0C6","website":"https://www.stakelab.fr","security_contact":"security@stakelab.zone","details":"Staking and relaying Hub for Cosmos ecosystem"},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1jp8na5d8p5m60mdpkh8tywjz62p7awq66wcmt0","validator_address":"dydxvaloper1jp8na5d8p5m60mdpkh8tywjz62p7awq6lswdq7","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"MNRvFNI0a4aO53YR/Bcrj5uR140oDPs1J00gcLy6BAs="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"8cbce559ecc5662c0a8bff68a4efe9fcadece143@65.109.112.162:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"ApNeeeaSpfuGe1sNFTXvBOJu6usy+YMnnJsDSLS65r+B"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["xav9gCj3Wj1t0J2G7x32pzBb0nbFWWQ4ASUGSyhle74Rk0Yqjd3KSwbUkqh8fdoEgLBV6QFS7BLJXb+2JyAxxA=="]}
+
```

Comment:
the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is `adv4tnt` and the self-delegation amount is `500000000000000000000000adv4tnt`

Reply:
Fixed.